diff --git a/home/base/tui/editors/packages.nix b/home/base/tui/editors/packages.nix
index 4deff377..8ed04e6d 100644
--- a/home/base/tui/editors/packages.nix
+++ b/home/base/tui/editors/packages.nix
@@ -35,7 +35,7 @@
       jdt-language-server
 
       #-- flutter
-      flutter
+      # flutter
 
       #-- dart
       # dart
@@ -205,7 +205,7 @@
     ++ [
       nodePackages.nodejs
       nodePackages.typescript
-      nodePackages.typescript-language-server
+      # nodePackages.typescript-language-server
       # HTML/CSS/JSON/ESLint language servers extracted from vscode
       nodePackages.vscode-langservers-extracted
       nodePackages."@tailwindcss/language-server"
@@ -216,11 +216,11 @@
       guile
       racket-minimal
       fnlfmt # fennel
-      (
-        if pkgs.stdenv.isDarwin
-        then pkgs.emptyDirectory
-        else pkgs-unstable.akkuPackages.scheme-langserver
-      )
+      #(
+      #  if pkgs.stdenv.isDarwin
+      #  then pkgs.emptyDirectory
+      #  else pkgs-unstable.akkuPackages.scheme-langserver
+      #)
     ]
     ++ [
       proselint # English prose linter
